doc/conflicts.md

```diff
@@ -10,7 +10,7 @@ MVCC data-model is used to do this. This page describes how it works.
 
 ## Replication
 
-The Sync datastore participates in master-less replication with
+The Sync document store participates in master-less replication with
```

Comment:
I think I just used `document store` partly because I was thinking in more abstract terms and I was probably influenced by the fact it was `datastore` and not `Datastore`. I think actually it makes sense to use `DocumentStore` as we are talking about the specific entity, so unless anyone disagrees I'll change it here so it's consistent with what you've done.